metastore/metastore-api/src/main/java/org/apache/drill/metastore/util/SchemaPathUtils.java

```diff
@@ -57,6 +57,8 @@ public static ColumnMetadata getColumnMetadata(SchemaPath schemaPath, TupleMetad
 
   /**
    * Adds column with specified schema path and type into specified {@code TupleMetadata schema}.
+   * For the case when specified {@link SchemaPath} has child, corresponding maps will be created
```

Comment:
Has one child? Or has children?

Reply:
You are right, children should be there, thanks, fixed.